source/game/g_teams.cpp

```diff
@@ -344,12 +344,14 @@ void SpawnTeams( RespawnQueues * queues ) {
 		}
 	}
 
+	bool even_teams = server_gs.gameState.gametype == Gametype_Gladiator || Cvar_Bool( "g_force_even_teams" );
```

Comment:
not in this pr

nuking it the first time around was a mistake too. should add more debug prints so we can figure out the bugs